examples/column_view_datagrid/grid_cell/imp.rs

```diff
@@ -9,7 +9,10 @@ use gtk::CompositeTemplate;
 #[template(file = "grid_cell.ui")]
 pub struct GridCell {
     #[template_child]
-    pub name: TemplateChild<gtk::Label>,
+    // gtk::Inscription requires gtk>=4.8. If you target an older version of gtk, can change to
```

Comment:
`gtk::Inscription requires gtk>=4.8. If you target an older version of gtk, you should switch to`